examples/kubernetes.json

```diff
@@ -5,7 +5,7 @@
       "orchestratorType": "Kubernetes"
     },
     "masterProfile": {
-      "count": 1,
+      "count": 3,

```

Comment:
why this change?

Reply:
sorry got caught up in the commit didnt mean to push it. just fixed it